src/beamlime/applications/daemons.py

```diff
@@ -1,63 +1,54 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright (c) 2024 Scipp contributors (https://github.com/scipp)
 from dataclasses import dataclass
-from typing import Any
+from typing import Any, AsyncGenerator, Generator
 
 from ._parameters import ChunkSize, DataFeedingSpeed
 from ._random_data_providers import RandomEvents
-from .base import DaemonInterface, MessageRouter
-from .handlers import Events, RawDataSent
+from .base import DaemonInterface, MessageProtocol
+from .handlers import Events
 
 
 @dataclass
-class BeamlimeMessage:
-    """A message object that can be exchanged between daemons or handlers."""
-
-    content: Any
-    sender: type = Any
-    receiver: type = Any
+class RawDataSent(MessageProtocol):
```

Comment:
Why does this class inherit from `MessageProtocol`? (`Stop` doesn't. And it's a protocol, not a base class.)

Reply:
That is true...! I think it's just I forgot to remove the inheritance after turning it into protocol....